src/platforms/apple/common/configuration/watchdog-terminations.mdx

```diff
@@ -46,3 +46,6 @@ SentrySDK.start { options in
     options.enableWatchdogTerminationTracking = NO;
 }];
 ```
+
+If you disable the `enableCrashHandler` option, the SDK disables the watchdog termination tracking as well, cause
+the SDK would report false positive watchdog terminations for each crash.
```

Comment:
```suggestion
If you disable the `enableCrashHandler` option, the SDK will disable watchdog termination tracking. This will prevent false positive watchdog termination reporting for every crash.
```